boards/arm64/fvp_base_revc_2xaemv8a/fvp_base_revc_2xaemv8a.dts

```diff
@@ -132,22 +132,27 @@
 			clocks = <&uartclk>;
 		};
 
-		eth: ethernet@1a000000 {
-			compatible = "smsc,lan91c111";
+		ethernet@1a000000 {
 			reg = <0x1a000000 0x1000>;
-			interrupts = <GIC_SPI 15 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
-			status = "disabled";
 
-			phy: phy {
-				compatible = "ethernet-phy";
+			eth: ethernet {
+				compatible = "smsc,lan91c111";
+				interrupts = <GIC_SPI 15 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
```

Comment:
doesn't look this binding is aligned with Linux, any reason why? https://elixir.bootlin.com/linux/v6.4.8/source/Documentation/devicetree/bindings/net/smsc,lan91c111.yaml#L4

Reply:
Does not look like the Linux one support specifying an mdio bus and phy. I could remove the parent node and specify the address on both nodes if you think that's better, that'll bring it closer but it's a bit silly to repeat it, this works well with how the Zephyr driver is written.